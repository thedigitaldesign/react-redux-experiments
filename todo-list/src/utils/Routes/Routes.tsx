import React, { Component } from 'react'

// Packages
import { Route } from 'react-router-dom'

// Pages
import Todo from '../../pages/Todo/Todo'
import User from '../../pages/User/User'

export default class Routes extends Component {
  render() {
    return (
      <>
        <Route path="/" exact component={User} />
        <Route path="/todo" exact component={Todo} />
      </>
    )
  }
}
