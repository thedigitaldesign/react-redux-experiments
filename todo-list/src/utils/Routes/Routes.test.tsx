import React from 'react'

// Packages
import Enzyme, {shallow} from 'enzyme'

// Routes
import Routes from './Routes'

describe('Routes', () => {
  const routes = shallow(<Routes />)

  it('should render routes', () => {
    expect(routes).toMatchSnapshot()
  });
});
