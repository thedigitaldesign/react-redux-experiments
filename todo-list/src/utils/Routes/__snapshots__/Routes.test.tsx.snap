// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render routes 1`] = `ShallowWrapper {}`;
